### ISSUE TITLE
uvicorn 0.27 support?

### ISSUE THREAD

**Issue by blalor** [Hannes221/rq-dashboard-fast#70]
Is it possible to add support for uvicorn > 0.26?  I'd like to add this package to an existing project that's using uvicorn 0.27.1 and Poetry can't resolve the dependencies:

    rq-dashboard-fast (>=0.3.1,<0.4.0) requires uvicorn (>=0.25.0,<0.26.0).

**Comment by blalor** [comment]
This update pinned the dependency on uvicorn to exactly 0.27.0; 0.27.1 is the latest. I think this should probably be `^0.27`, or even `>=0.27`.

**Comment by blalor** [comment]
I think this might be more permissive set of versions:
```
fastapi = "^0"
redis = "^5.0"
rq = "^1.16"
uvicorn = "^0.27"
```
I'm not sure how to best define dependencies for running the app standalone *and* making it installable as a library. Maybe separate groups?

**Comment by Hannes221** [comment]
Thanks for pointing out!

**Comment by Hannes221** [comment]
Should be resolved in v0.3.6
